lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVM.cpp

```diff
@@ -593,8 +588,38 @@ class ConvertTritonGPUOpToLLVMPattern
             multiDimNanoTileId[k] *
                 (sizePerThread[k] * threadsPerWarp[k] * warpsPerCTA[k]) +
             multiDimNanoTileElemId[k];
-        multiDimIdx[n][k] =
-            add(multiDimBase[k], idx_val(offset[k][reorderedMultiDimId]));
+        reordered_offset[n].push_back(offset[k][reorderedMultiDimId]);
+      }
+    }
+    return reordered_offset;
```

Comment:
Note the variable name should be camel case, I have a detailed comment below.

Reply:
done